### DISCUSSION TITLE
Tooljet connecting to MinIO

### DISCUSSION THREAD

**Discussion by Gunkev** [minio/minio#15111] [Q&A] [answered]
Hello I want to connect tooljet to MinIO.  I ran min.exe server in my Windows OS everything seems to working well. I was able to connect and create my buckets. But when I try to connect to Tooljet I get the error `invalid port:9001`.  My console is running on http://127.0.0.1:9001
![errorMin](https://user-images.githubusercontent.com/36812576/174477987-16fc88b4-22fc-4f76-9878-1c3104f8c292.PNG)

**Comment by harshavardhana** [comment, marked as answer]
Tooljet needs S3 URL not console URL use port 9000

**Reply by Gunkev** [reply]
> Tooljet needs S3 URL not console URL use port 9000

@harshavardhana how do I obtain s3 url ? . Actually I'm using MinIO as data-source. Is S3 not only for amazon ?

**Reply by klauspost** [reply]
Let's call it the S3 API URL then. My guess is that it is `http://127.0.0.1:9000/` on your system, but it will be shown at startup.

**Reply by Gunkev** [reply]
> Let's call it the S3 API URL then. My guess is that it is `http://127.0.0.1:9000/` on your system, but it will be shown at startup.

@klauspost yes it is` http://127.0.0.1:9000/` host is `127.0.0.1` and port `9000` but it returns the same error when I change and used 9000

**Reply by ravindk89** [reply]
If you can connect with the MinIO CLI `mc`, then this might be a ToolJet issue (looks like you are following [their docs](https://docs.tooljet.com/docs/data-sources/minio)

```shell
mc alias set local http://localhost:9000 ACCESS SECRET
mc ls
mc admin info
```

If those work, then its not a connectivity issue. If ToolJet is not running on your local machine, then you cannot specify localhost as the address.

**Reply by klauspost** [reply]
I advise you get in touch with the tooljet authors. In particular it sounds interesting why "9000" isn't a valid port.

**Comment by Gunkev** [comment]
Yeah I have already contacted the Tooljet maintainers

Le lun. 20 juin 2022 à 17:18, Klaus Post ***@***.***> a
écrit :
